### ISSUE TITLE
Bug: s3 storage/minio : pdf file created when we delete file in tracim (odf file)

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#4001]
## Description and expectations

Related to #4000 

When I delete my odp file, Tracim create a new pdf preview of this deleted file.
- its not necessary to create pdf file on delete
- pdf created increase space used for a deleted file

Existing file in preview folder
~~~
total 144M
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:00 2cbd2ed344591899e76bfd43a1cab403-500x500-page0
-rw-rw-rw- 1 www-data www-data 33K déc.  30 10:00 2cbd2ed344591899e76bfd43a1cab403-500x500-page0.jpeg
-rwxrwxrwx 1 www-data www-data   0 déc.  30 10:00 2cbd2ed344591899e76bfd43a1cab403.lock
-rw-rw-rw- 1 www-data www-data   2 déc.  30 10:00 2cbd2ed344591899e76bfd43a1cab403_page_nb
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:00 2cbd2ed344591899e76bfd43a1cab403.pdf
-rw-rw-rw- 1 www-data www-data   2 déc.  30 10:00 58870ca566246a789e3ff7998c026cfc_page_nb
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:00 58870ca566246a789e3ff7998c026cfc.pdf
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:06 59043e4cd2b1f9ba0596ebf7846c49e9-500x500-page0
-rw-rw-rw- 1 www-data www-data 33K déc.  30 10:06 59043e4cd2b1f9ba0596ebf7846c49e9-500x500-page0.jpeg
-rwxrwxrwx 1 www-data www-data   0 déc.  30 10:06 59043e4cd2b1f9ba0596ebf7846c49e9.lock
-rw-rw-rw- 1 www-data www-data   2 déc.  30 10:06 59043e4cd2b1f9ba0596ebf7846c49e9_page_nb
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:06 59043e4cd2b1f9ba0596ebf7846c49e9.pdf
~~~
This is the new pdf created
~~~
-rw-rw-rw- 1 www-data www-data   2 déc.  30 10:13 5f065a22ddd2b5a3c6b95430e6d47663_page_nb
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:13 5f065a22ddd2b5a3c6b95430e6d47663.pdf
~~~
Existing file in preview folder
~~~
-rw-rw-rw- 1 www-data www-data   2 déc.  30 09:59 7acd9736718ebe72199db0603ae2c739_page_nb
-rw-rw-rw- 1 www-data www-data 16M déc.  30 09:59 7acd9736718ebe72199db0603ae2c739.pdf
-rw-rw-rw- 1 www-data www-data   0 déc.  30 10:00 7de99d4620693e2bf273955d80c39f29.pdf_flag
-rw-rw-rw- 1 www-data www-data 21M déc.  30 10:00 7de99d4620693e2bf273955d80c39f29.pdf.odp
-rw-rw-rw- 1 www-data www-data   2 déc.  30 10:00 f2726df29616e2304e5f92e190224743_page_nb
-rw-rw-rw- 1 www-data www-data 16M déc.  30 10:00 f2726df29616e2304e5f92e190224743.pdf
-rwxrwxrwx 1 www-data www-data   0 déc.  30 10:12 libreoffice.lock
~~~

### How to reproduce

- delete one file in tracim directly from all contents list
=> you can seen one pdf file created in preview folder. This is the preview of deleted file.

### Version information

- Tracim version: [e.g. v2.7, Mon Tracim]
- Web browser: [e.g. Google Chrome, Firefox, Safari]
- Version: [e.g. 22, latest]
- Device / System: [e.g. PC, iPhone 6]

## Screenshots

________________________________________________

**Comment by inkhey** [comment]
@PhilippeAccorsi When you say "delete", you mean using frontend ui ?

I see 2 reasons preview may be generated for deleted file 
- Frontend ask to know the preview of deleted file, this is logical as frontend stay on deleted file page after clicking on delete button. That's why jpg preview is generated
- Tracim Live message or standard content endpoint contain page number for office document. Current mecanism to get page number of some kind of file (office file generated using libreoffice) need creation of preview first: https://github.com/algoo/preview-generator/blob/e61675f276e1375d82ef8f2f27c017cf7d0e8272/preview_generator/preview/builder/document_generic.py#L166 .

Not sure how we can improve this effectively.

**Comment by PhilippeAccorsi** [comment]
> @PhilippeAccorsi When you say "delete", you mean using frontend ui ?

I mean using frontend but without app file open.

**Comment by PhilippeAccorsi** [comment]
> Not sure how we can improve this effectively.

I understand some of point but not really interst of creating pdf file for a deleted file when user don't try to seen the deleted file.

**Comment by inkhey** [comment]
> > Not sure how we can improve this effectively.
> 
> I understand some of point but not really interst of creating pdf file for a deleted file when user don't try to seen the deleted file.

I created an issue about finding a more cleaner way to separate preview feature from standard content feature, which avoid to create preview when it's not useful:  https://github.com/tracim/tracim/issues/4010

**Comment by inkhey** [comment]
this issue will not be fixed by itself, may be solved by #4010.
